src/app/page/UserProfile/UserProfile.js

```diff
@@ -35,14 +39,14 @@ const useStyles = makeStyles((theme) => ({
 //   };
 // }
 
-const ProfileControlButtons = ({ isEdit, saveAction, cancelAction, editAction }) => {
+const ProfileControlButtons = ({ isEdit, saveAction, cancelAction, editAction, classes }) => {
   return isEdit ? (
     <>
-      <Button size="large" onClick={saveAction}>
+      <Button size="large" onClick={saveAction} className={classes.controlButtons}>
```

Comment:
Can we instead do this:
`<Grid container spacing={1}>
     <Grid item><Button/></Grid>
     <Grid item><Button/></Grid>
</Grid>`

This way, we let MUI handle all of the responsive styling.

Reply:
Yes definitely thanks for the solution! I'll put up an update soon

Reply:
There are also merge conflicts.

Reply:
Alright I resolved merge conflicts and have updated code on the source branch